sql/core/src/test/scala/org/apache/spark/sql/errors/QueryExecutionErrorsSuite.scala

```diff
@@ -398,4 +398,32 @@ class QueryExecutionErrorsSuite
         "\\(QueryExecutionErrorsSuite\\$\\$Lambda\\$\\d+/\\w+: \\(string, int\\) => string\\)",
       matchMsg = true)
   }
+
+  test("INCOMPARABLE_PIVOT_COLUMN: Pivot columns can't be comparable") {
```

Comment:
Maybe, the columns can't be incomparable?
```suggestion
  test("INCOMPARABLE_PIVOT_COLUMN: an incomparable column of the map type") {
```